README.md

```diff
@@ -1,3 +1,20 @@
-# Backend app
+# Backend Music Library
+
+## Description
+
+Provide a short description explaining the what, why, and how of your project. Use the following questions as a guide:
+
+I built this project in order to expand my knowledge on jest and Javascript, and to put my skills to the test with what I have learnt through my course.
+My music library solves the problem of having a neat and convenient way to store your favourite artists, and it allows you and others to see more about that artsit and the albums they have released.
+From this project I have learnt how to use Postman and the API's to add artists and albums. I have also learnt more about express, and the syntax that comes with it.
```

Comment:
It would be good to include a more comprehensive list of the technologies used in the project including the dependencies and dev dependencies (found in package.json)